internal/handlers/auth/github.go

```diff
@@ -40,7 +44,12 @@ func (a *githubAdapter) GetUserData(state, code string) (*user, error) {
 	if err != nil {
 		return nil, errors.Wrap(err, "could not exchange code")
 	}
-	oAuthUserInfoReq, err := a.config.Client(context.Background(), oAuthToken).Get("https://api.github.com/user")
+
+	gitHubUserURL := "https://github.homedepot.com/api/v3/user"
```

Comment:
homedepot.com seems unlikely to be a reasonable default :) :) :)

Reply:
done